includes/admin/extensions/abstract-extension.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace EDD\Admin\Extensions;
+
+use \EDD\Admin\Pass_Manager;
+use \EDD\Admin\Extensions\ExtensionsAPI;
```

Comment:
```suggestion
```
Not needed, as it never gets used. (`ExtensionsAPI` is referenced, but they're in the same namespace so it doesn't need to be imported.)